src/tests/sentry-mocked.test.ts

```diff
@@ -123,4 +130,46 @@ Object {
             },
         });
     });
+
+    test('can pass sentry tags', async () => {
+        const dateNow = Date.now;
+        Date.now = jest.fn(() => 1520343036000);
+        await new Promise((resolve, reject) => {
+            const logger = loggerFactory();
+            extendSentry(logger, { sentry: 'DSN' });
+            captureMessage.mockImplementation(createCapture(resolve));
+            logger.fatal('sentryfatal', (scope: Scope) => {
```

Comment:
I think typing/overrides are almost essential here to avoid confusion with `cosmas.log({...}, "msg")`

Reply:
Added